exercises/concept/blackjack/.docs/instructions.md

```diff
@@ -62,7 +62,7 @@ Depending on your two cards and the card of the dealer, there is a strategy for
 Although not optimal yet, you will follow the strategy your friend Alex has been developing, which is as follows:
 
 - If you have a pair of aces you must always split them.
-- If you have a Blackjack (two cards that sum up to a value of 21), and the dealer does not have an ace, a figure or a ten then you automatically win.
+- If you have a Blackjack (two cards that sum up to a value of 21), and the dealer does not have an ace, a face card (Jack, Queen, King) or a ten then you automatically win.
```

Comment:
```suggestion
- If you have a Blackjack (two cards that sum up to a value of 21), and the dealer does not have an ace , 10, or a face card (Jack, Queen, King) then you automatically win.
```

The 10 could be, and probably should be, mentioned first.

Reply:
Changing "ten" to "10" would be inconsistent with the rest of file.

Reply:
Also, this work is more than what is claimed in the discussion, as we have not only changed the word from "figure" to "face card" but have given a list of face cards here.

Reply:
You are correct, it was here to talk about, not to commit directly.  The names we are using as names and values as values, we definitely should keep doing that.